README.md

```diff
@@ -30,7 +30,7 @@ the Rust programming language. It is:
 
 [Website](https://tokio.rs) |
 [Guides](https://tokio.rs/docs/) |
-[API Docs](https://docs.rs/tokio/0.2.0-alpha.2/tokio) |
+[API Docs](https://docs.rs/tokio/latest/tokio) |
```

Comment:
could probably just link to `docs.rs/tokio` though I think this achieves the same thing anyways.